simapp/go.mod

```
@@ -187,10 +188,7 @@ require (
 )
 
 // This can be deleted after the CometBFT PR is merged
```

```suggestion
// This can be deleted after the CometBFT PR is merged
```
should we delete this comment ?

---

Wrong comment, thanks!